src/openrct2/drawing/Drawing.String.cpp

```diff
@@ -837,8 +837,11 @@ static void ttf_process_string_literal(rct_drawpixelinfo* dpi, std::string_view
                 if (ttfRunIndex.has_value())
                 {
                     // Draw the TTF run
+#    pragma GCC diagnostic push
+#    pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
```

Comment:
If this works, add a comment explaining it’s for https://github.com/OpenRCT2/OpenRCT2/issues/17371 and specify a date at which to re-evaluate.